src/modules/portfolio/portfolio-item.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="portfolio__item">
+  <div class="portfolio__preview">
+    <img class="portfolio__preview-logo" src="{{ logo }}">
+    <div class="portfolio__preview-background {{ previewBackgroundClass }}"><img class="portfolio__preview-image {{ portfolioPreviewClass }}" src="{{ device }}"></div>
+  </div>
+  <div class="portfolio__info">
+    <div class="portfolio__info-title">{{ title }}</div>
+    <div class="portfolio__info-description">{{ info_description }}</div>
+  </div>  
+  <div class="portfolio__devices">
+    <div class="portfolio__devices-nav">
+      <span class="portfolio__devices-item">
+        <a href="#" class="portfolio__devices-link">Desktop</a>
+      </span>
+      <span class="portfolio__devices-item portfolio__devices-item_space">
+        <a href="#" class="portfolio__devices-link">Tablet</a>
+      </span>
+      <span class="portfolio__devices-item">
+        <a href="#" class="portfolio__devices-link">Mobile</a>
+      </span>
+    </div>
+    <div class="portfolio__devices-description">{{ devices_description }}</div>
+    <div class="portfolio__devices-image">
+      <img src="{{ screenshot }}" class="portfolio__devices-screenshot">
```

Comment:
is it same img for  all of this?

Reply:
it is the same in this file, but different in data.js
screenshot: "static/img/screenshot_1.png",
screenshot: "static/img/screenshot_2.png",

Reply:
ok